README.md

```diff
@@ -1,2 +1,4 @@
 # opendatahub-tests
 # TODO: update file
+
+# Run `pre-commit` before committing. You will need npm installed to do so.
```

Comment:
Should be under `CONTRIBUTING.md` with:
## Code check
Install pre-commit for code check.
```bash
pre-commit install
```

Reply:
https://github.com/opendatahub-io/opendatahub-tests/pull/25